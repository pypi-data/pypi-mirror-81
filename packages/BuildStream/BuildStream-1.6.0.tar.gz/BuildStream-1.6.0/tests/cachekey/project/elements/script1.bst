kind: script
depends:
- filename: elements/import1.bst
  type: build

config:
  commands:
  - echo "Hello World !"
