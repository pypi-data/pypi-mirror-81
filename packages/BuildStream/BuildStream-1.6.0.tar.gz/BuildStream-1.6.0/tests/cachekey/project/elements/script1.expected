7e4e7e8ec36d9b661b357637cb4d2fb9adc37481764d645382d6b0ccb6ad7c71